snapcraft/internal/pluginhandler/__init__.py

```diff
@@ -146,7 +146,7 @@ def _load_code(self, plugin_name, properties, part_schema,
         plugin = _get_plugin(module)
         if not plugin:
             raise errors.PluginError(
-                'no plugin found in module: {}'.format(plugin_name))
+                'no plugin found in package: {}'.format(plugin_name))
```

Comment:
This error message (which is being added in this PR) is hit in two cases: there's a module with the name of the plugin being loaded, or a package with the name of the plugin being loaded, and something inheriting from `BasePlugin` is not found in them. Either language is probably too specific to work for both cases, so I changed it to `no plugin found in {!r}`.